index.html

```diff
@@ -1,19 +1,105 @@
-<!doctype html>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-
-  <title>My Resume</title>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
-  <link rel="stylesheet" href="css/index.css">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+     <link rel = "stylesheet"
+        type = "text/css"
+        href = "C:\Git\Preprocessing-I\css\index.css" />
```

Comment:
```suggestion
        href = "/css/index.css" />
```
This path is too specific, I couldn't view your page on htmlpreview, or when I cloned the project until I changed this path. Usually you can just reference the path in relation to the file you are in. So from index.html you have to go into the css directory to the index.css file, so /css/index.css